src/rust/grapl-web-ui/tests/api/config.rs

```diff
@@ -0,0 +1,56 @@
+use grapl_config::env_helpers::FromEnv;
+use rusoto_dynamodb::DynamoDbClient;
+
+type Result<T> = std::result::Result<T, TestConfigError>;
+
+pub struct TestConfig {
```

Comment:
fwiw, in the rest of our codebase we've mostly been using Clap for inputting env var stuff like this, but no biggie either way

Reply:
Yeah, that's a good point - it's what I should have done. I think I'll make that a separate PR after this, where I'll migrate this and the main app to clap as well.

I migrated the main app to use clap in a later draft PR (https://github.com/grapl-security/grapl/pull/1839) - I'll start by bringing that part into its own.